app/views/warnings/_scheme_12_fees_summary.html.haml

```diff
@@ -0,0 +1,8 @@
+- if claim&.last_injection_attempt&.succeeded && claim.has_scheme_12_fees?
+  .js-callout-scheme-12_fees-warning
+    .warning-summary{ role: 'group', 'aria-labelledby': 'warning-summary-heading', tabindex: '-1' }
+      %h2.heading-medium.warning-summary-heading{ id: 'warning-summary-heading' }
+        = t('shared.injection_errors.scheme_12_warning.header')
+
+      %span.form-hint
+        = t('shared.injection_errors.scheme_12_warning.message')
```

Comment:
consistent naming: `scheme_12_warning` -> `scheme_12_fees_warning`

Reply:
👍 have amended naming to scheme_12_fees_warning